stopZookeeper.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sudo /opt/apache-zookeeper-3.8.4-bin/bin/zkServer.sh stop
```

Comment:
These paths are only going to work for zk installed in that dir. Use ZOOKEEPER_HOME, or make an env file

Reply:
will fix